Telegram/README.md

```diff
@@ -0,0 +1,46 @@
+# Telegram Integration Prototype
+
+
+### Prerequisites
+
+- Follow the instructions on the [main README file](https://github.com/GoogleCloudPlatform/dialogflow-integrations#readme) in the root directory of this repository.
```

Comment:
In the main readme, there is a section to set INTEGRATION environment, as far as I remember, that is used to determine which folder docker image should be using. For our case, the folder becomes /cx/INTEGRATION, do we also need to update that section?